srcpkgs/wayfire-plugins-extra/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'wayfire-plugins-extra'
+pkgname=wayfire-plugins-extra
+version=0.0.1
+revision=1
+build_style=meson
+hostmakedepends="pkg-config git"
+makedepends="wayfire-devel glibmm-devel"
+depends=""
+short_desc="Additional plugins for Wayfire"
+maintainer="Érico Nogueira <ericonr@disroot.org>"
+license="MIT"
+homepage="https://wayfire.org/"
+#distfiles="https://github.com/WayfireWM/wayfire-plugins-extra/archive/${version}.tar.gz"
```

Comment:
Use specific commit in `distfiles`, drop `git` from `hostmakedepends` and drop `do_fetch`.

Also: I pinged the "Release" issue over at `wayfire-plugins-extra` to hopefully get the maintainers to drop the release.

Reply:
It is going to get a release along with this batch, the main maintainer didn't do it yet. The temporary git versions is for this PR, when merged into void-packages it will be the released version.